conda_forge_tick/migrators.xsh

```diff
@@ -822,6 +822,27 @@ class CompilerRebuild(Rebuild):
                             self.graph.out_edges($PROJECT))[:5]])))
         return body
 
+class OpenSSLRebuild(Rebuild):
+    migrator_version = 1
+
+    def migrate(self, recipe_dir, attrs, **kwargs):
```

Comment:
Why is this needed?

Reply:
hmm yeah, it just seems to be calling the super

Reply:
We might not need the class. I think the message might be the stock rebuild message?

Reply:
Well the message probably should be customized

Reply:
Ya, I copy-pasted this from the CompilerRebuild class. I figure that they are logically different rebuilds so they should get different classes. Any ideas on what the rebuild message should be? I was thinking to add something at the beginning of the message like "This PR has been triggered in an effort to update openSSL."

Reply:
yeah, I think that is good enough.  And I agree they are logically different and should get different version numbers

Reply:
I think this should be different from the `CompilerRebuild` but maybe not from the `Rebuild` general class. To some extent this may be a moot point for our current level of development, but I'm not certain that we want a bespoke class per rebuild migration. The compiler and arch got their own because they need to poke other parts of the CF infrastructure (changing the yaml files) than just re-rendering and bumping the build number.

Reply:
Hmmm, I think it is useful to track different package ABI breakages separately.  Right now, that is done by having a minimal subclass with its own version number.  Also, this gives the option of heavily customizing the rebuild if needed.